constants.bal

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+//Base URL
+const string BASE_URL = "https://www.googleapis.com";
+
+// URL encoding
+const string ENCODING_CHARSET = "utf-8";
+
+//Symbols
+const string QUESTION_MARK = "?";
+const string PATH_SEPARATOR = "/";
+const string EMPTY_STRING = "";
+const string WHITE_SPACE = " ";
+const string FORWARD_SLASH = "/";
+const string DASH_WITH_WHITE_SPACES_SYMBOL = " - ";
+const string COLON = ":";
+const string EXCLAMATION_MARK = "!";
+const string EQUAL = "=";
+const string _ALL = "*";
+const string TRUE = "true";
+const string FALSE = "false";
+
+// Drive
+const string DRIVE_URL = "https://www.googleapis.com";
+const string REFRESH_URL = "https://www.googleapis.com/oauth2/v3/token";
```

Comment:
As a suggestion we can make the REFRESH_URL public so that, the user of the connector can use it as `drive:REFRESH_URL` . Please clarify with others working on google related connectors.

Reply:
Yeah. Its a good tip. changed to public